demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -1015,6 +1012,23 @@ def validate_modeling_rule(
             extra={"markup": True},
         )
         test_data = TestData.parse_file(modeling_rule.testdata_path.as_posix())
+        modeling_rule_is_compatible = validate_modeling_rule_version_against_tenant(
+            to_version=modeling_rule.to_version,
+            from_version=modeling_rule.from_version,
+            tenant_demisto_version=tenant_demisto_version,
+        )
+        if not modeling_rule_is_compatible:
+            # Modeling rule version is not compatible with the demisto version of the tenant, skipping
+            skipped = f"XSIAM Tenant's Demisto version doesn't match Modeling Rule {modeling_rule} version, skipping"
+            logger.warning(f"[yellow]{skipped}[/yellow]", extra={"markup": True})
```

Comment:
should it be warning? maybe change to debug because we simply skip as its incompatible

Reply:
We thought of keeping it as a **warning** so it can be shown in the gitlab step's UI, I believe if it is debug, then we would have to access the artifacts. WDYT?

Reply:
lets keep it warning